<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>601</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
    </property>
    <property name="text">
     <string>Signup Seller</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>380</y>
      <width>601</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
    </property>
    <property name="text">
     <string>Signup Customer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>__________________  Are You Already A Customer Or Seller?______________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>621</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>___________________________Are You A New Seller?__________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>___________________________Are You A New Customer?__________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lobster</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>Connect Us:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>63</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>63</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>63</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>63</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic_label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>63</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>861</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt;
font: 28pt &quot;Snap ITC&quot;;
color: rgb(77, 119, 255);
font-weight: 900;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Welcome To FCIS-MARKET :))</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
